.ebextensions/05_nginx_proxy.config

```diff
@@ -0,0 +1,65 @@
+files:
+  "/etc/nginx/sites-available/000-default.conf":
+    mode: "000644"
+    owner: root
+    group: root
+    content: |
+      map $http_upgrade $connection_upgrade {
+        default   "upgrade";
+        ""        "";
+      }
+
+      server {
+        listen         80;
+        server_name    $ENV_URL;
+
+        gzip on;
+        gzip_comp_level 4;
```

Comment:
@Tuuleh do you know  if `gzip on` is present in eb's default nginx config? Or are we enabling it for the first time?